providers/nitro/nitro-enclave/src/main.rs

```diff
@@ -1,17 +1,33 @@
 mod nitro;
 use nix::sys::socket::SockAddr;
-use tracing::{error, info, warn, Level};
-use tracing_subscriber::FmtSubscriber;
+use tmkms_light::tracing_subscriber::Layer;
+use tmkms_nitro_helper::VSOCK_HOST_CID;
+use tracing::{error, info, warn};
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::{fmt, EnvFilter};
 use vsock::VsockListener;
 
 fn main() {
-    // TODO: subscriber for production to forward to vsock
+    let log_server_port = std::env::args()
+        .next()
+        .map(|x| x.parse::<u32>().ok())
+        .flatten()
+        .unwrap_or(6050);
+    let layer = Layer::new(VSOCK_HOST_CID, log_server_port);
+    let fmt_layer = fmt::layer().with_target(false);
+
     // TODO: the log level via a command line arg?
-    let subscriber = FmtSubscriber::builder()
-        .with_max_level(Level::INFO)
-        .finish();
```

Comment:
The default subscriber may still be useful in development / debug mode where one can access the console -- maybe the enclave can take an optional parameter that if provided, it'd use `FmtSubscriber` (perhaps with a higher log level?) -- in that case, one would also need to be able to disable log_server in the helper?

Reply:
in the new subscriber, there are tree layers: the `fmt_layer`(which print the log into the consle), the `filter layer`(which control the  log level) and the `vsock layer`(which push log into vsock server).  So, the new subscriber can print log into the console and also push log into vsocket, and in my test I can see the log in the `nitro-cli console ` command.

Reply:
ok, that works too